scripts/Set-AllUserLocalesToUk.ps1

```diff
@@ -0,0 +1,39 @@
+Install-Module -Name Microsoft.Xrm.Tooling.CrmConnector.PowerShell -Force -Scope CurrentUser -AllowClobber
+
+Write-Host "Updating user settings to English (United Kingdom)"
+
+$connectionString = "AuthType=OAuth; Username=$env:username; Password=$env:password; Url=$env:environmentUrl; AppId=51f81489-12ee-4a9e-aaae-a2591f45987d; RedirectUri=app://58145B91-0C36-4500-8554-080854F2AC97; LoginPrompt=Auto";
```

Comment:
Might be best to change this to client ID and secret.

Reply:
Client Id and secret has now replaced the username & password